apps/web/components/Communities.tsx

```diff
@@ -0,0 +1,89 @@
+"use client";
+import React, { useEffect, useState } from "react";
+import { useAccount } from "wagmi";
+import { CommunityCard } from "./CommunityCard";
+import { RegistryCommunity } from "#/subgraph/.graphclient";
+
+export function Communities({
+  communities,
+}: {
+  communities: RegistryCommunity[];
+}) {
+  const { address } = useAccount();
+
+  const [otherCommunities, setOtherCommunities] =
+    useState<RegistryCommunity[]>(communities);
+  const [userCommunities, setUserCommunities] = useState<RegistryCommunity[]>(
+    [],
+  );
+
+  useEffect(() => {
+    const auxOtherCommunities: RegistryCommunity[] = [];
+    const auxUserCommunities: RegistryCommunity[] = [];
+    for (let community of communities) {
+      if (memberInCommunity(community)) {
+        auxUserCommunities.push(community);
+      } else {
+        auxOtherCommunities.push(community);
+      }
+    }
+    setUserCommunities(auxUserCommunities);
+    setOtherCommunities(auxOtherCommunities);
+  }, [address]);
+
+  function memberInCommunity(community: RegistryCommunity) {
+    if (!community?.members) {
+      return false;
+    }
+    for (let member of community?.members) {
+      if (member?.memberAddress?.toLowerCase() == address?.toLowerCase()) {
```

Comment:
I think you have some problem with linter, it doesn't usually allow double ==

Reply:
I didn't know linter should prevent this. We don't need js to perform type coercion here so i will change it.